resources/js/Pages/EventPage.vue

```diff
@@ -140,106 +121,72 @@ const isPaid = true
+                :line2="event.address"
               />
 
-              <div
-                class="w-full flex flex-col min-[467px]:flex-row justify-between gap-4"
-              >
+              <div class="w-full flex justify-between gap-4">
                 <InfoBlock
-                  :icon="UserIcon"
-                  title="Piotr Nowak"
-                  line1="Organiztor"
+                  :image-url="event.owner?.avatar_url || event.owner?.group_url || ''"
+                  :title="`${event.owner?.first_name?? ''} ${event.owner?.last_name?? ''} ${event.owner?.name ?? ''}`.trim() ?? 'Nieznany'"
+                  :line2="event.owner_type"
                 />
-                <div class="flex items-center justify-start sm:justify-end">
-                  <button class="bg-brand/10 text-brand px-3 py-1 rounded-xl">
-                    Obserwuj
-                  </button>
+                <div class="flex items-center justify-end">
+                  <button class="bg-brand/10 text-brand px-3 text-sm sm:text-base py-1 rounded-xl">Obserwuj</button>
                 </div>
               </div>
 
-              <h1 class="font-medium text-3xl text-[#120D26]">Informacje</h1>
-              <p class="font-normal text-xl text-[#120D26]">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit Lorem
-                ipsum dolor sit amet, consectetur adipiscing elit Lorem ipsum
-                dolor sit amet, consectetur adipiscing elit Lorem ipsum dolor
-                sit amet, consectetur adipiscing elit Lorem ipsum dolor sit
-                amet, consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit Lorem ipsum dolor sit amet,
-                consectetur adipiscing elit
-              </p>
+              <h1 class="font-medium sm:text-3xl text-xl text-[#120D26]">Informacje</h1>
+              <p class="font-normal sm:text-xl text-sm text-[#120D26]">{{ event.description }}</p>
             </div>
           </div>
 
-          <div class="flex flex-col space-y-6 md:w-2/6 justify-start">
-            <div
-              class="sm:hidden fixed bottom-0 left-0 w-full z-[1001] bg-white p-4 shadow-t"
-            >
-              <base-button
-                class="w-full text-base h-16 p-[15px] bg-brand-dark text-white rounded-2xl"
-              >
-                <span
-                  class="inline-flex font-semibold items-center justify-center space-x-2"
-                >
+          <div class="flex flex-col space-y-6 lg:w-2/6 justify-start">
+            <div v-if="event.is_paid" class="sm:hidden fixed bottom-0 left-0 w-full z-[1001] bg-white p-4 shadow-t">
```

Comment:
I think you don’t need such a large value for `z-index`

```suggestion
            <div v-if="event.is_paid" class="sm:hidden fixed bottom-0 left-0 w-full z-40 bg-white p-4 shadow-t">
```